packages/uikit/src/tokens/colors.ts

```diff
@@ -45,6 +45,12 @@ export const lightColors = {
   textDisabled: "#BDC2C4",
   textSubtle: "#7A6EAA",
   disabled: "#E9EAEB",
+  primary10: "#EEFBFC",
+  primary60: "#02919D",
+  positive10: "#EAFBF7",
+  positive60: "#129E7D",
+  destructive10: "#FFF0F9",
+  destructive: "#ED4B9E",
```

Comment:
does these variant can use global?

Reply:
i think so, we can treat them as the new design token. variant is the context in Tag

Reply:
dose this colors have added to figma design system?

Reply:
yeah, they are from figma